src/Features/Core/Portable/SplitOrMergeIfStatements/IIfLikeStatementGenerator.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.SplitOrMergeIfStatements
+{
+    /// <summary>
+    /// An if-like statement is either an if statement or an else-if clause.
+    /// When querying the syntax, C# else if chains are "flattened" and modeled to look like VB else-if clauses.
+    /// </summary>
```

Comment:
i really like this class.  Very cool.  Certain methods on it could use explanations of what they're doing.

Reply:
➡️ Fixed in 81366a4eba507c5192ecfb911cdd8ef832c0d996